### PR TITLE
feat(website): add query playground

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: crates/oxc_query/src/adapter.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ pub struct Adapter<'a> {
     pub path_components: Vec<Option<String>>,
 }
 
-const SCHEMA_TEXT: &str = include_str!("./schema.graphql");
+pub const SCHEMA_TEXT: &str = include_str!("./schema.graphql");
```
Comment (u9g) [inline crates/oxc_query/src/adapter.rs:26]: Is it a bad idea to expose our schema text as part of the crate? I don't want to give any guarantees on it...

Comment (Boshen) [reply crates/oxc_query/src/adapter.rs:26]: This is an internal crate so is fine I think.

File: website/patches/graphql-language-service@5.1.7.patch [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+-        return getSuggestionsForDirective(token, state, schema, kind);
++        // We should autocomplete fields on the line after we autocomplete a directive
++        if (state.needsAdvance) { // active after '@' or many other kinds of punctuation, so perfect for when we need to autocomplete directives
++            return getSuggestionsForDirective(token, state, schema, kind);
++        } else { // there has been no '@' so we should autocomplete fields
++            return getSuggestionsForFieldNames(token, typeInfo, opts);
++        }
+     }
+     return [];
+ }
```
Comment (Boshen) [inline website/patches/graphql-language-service@5.1.7.patch:18]: lol wat? TIL you can do this.

**Review by Boshen — APPROVED** [review]
playground.js is getting big, I wonder if we should encapsulate some of the logic into another Class.

I approved, so you decide on either merge or refactor.

File: crates/oxc_query/src/adapter.rs (hunk 1 above)
Comment (u9g) [reply crates/oxc_query/src/adapter.rs:26]: Sounds good

File: website/patches/graphql-language-service@5.1.7.patch (hunk 2 above)
Comment (u9g) [reply website/patches/graphql-language-service@5.1.7.patch:18]: Yep, I'll aim to fix every bug I find in our dependencies that effects :)

**Pull request merged**